src/utils/session-core.js

```diff
@@ -683,33 +685,52 @@ export const updateUserActivity = async (session) => {
  * @function sendMessageTTL
  * @param {Session} session - Solid's Session Object {@link Session}
  * @param {object} messageObject - An object containing inputs for the the message
- * @param {string} otherPodUsername - The username of the other user you're sending a message to
  * @returns {Promise} Promise - Updates last active time of user to lastActive.ttl
  */
 
-export const sendMessageTTL = async (session, messageObject, otherPodUsername) => {
-  const containerUrl = getContainerUrl(session, 'Inbox', 'cross-fetch', otherPodUsername);
+export const sendMessageTTL = async (session, messageObject) => {
+  const { title, message, recipientUsername } = messageObject;
+  const containerUrl = getContainerUrl(session, 'Inbox', 'cross-fetch', recipientUsername);
+  const inboxUrl = getContainerUrl(session, 'Inbox', 'self-fetch');
+
   const senderUsername = session.info.webId.split('profile')[0].split('/')[2].split('.')[0];
+  const recipientWebId = `https://${recipientUsername}.${
```

Comment:
This should probably be an input field. Solid doesn't enforce that the WebId URI have any particular format or access protocol. We also realistically won't be sending messages only to pods stored at our specific identity provider.